api-tester/src/main/java/com/revenuecat/apitester/java/PurchasesAPI.java

```diff
@@ -184,6 +186,7 @@ static void check(final Purchases purchases, final Map<String, String> attribute
         purchases.setCreative("");
     }
 
+    @OptIn(markerClass = ExperimentalPreviewRevenueCatPurchasesAPI.class)
```

Comment:
This is concerning, since in Java, this is only caught by the android linter, not at compile time, so it's possible that Java users use the API inadvertently.